### PR TITLE
Add page about Query Log Analyzer

### PR DESCRIPTION
**Pull request by DavidJsson**

### CODE REVIEW

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-aura-382.surge.sh

New pages:
- [Query log analyzer](https://neo4j-docs-aura-382.surge.sh/aura/platform/logging/query-log-analyzer)

File: modules/ROOT/pages/platform/logging/query-log-analyzer.adoc [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+. Navigate to the https://console.neo4j.io/?product=aura-db[Neo4j Aura Console] in your browser.
+. Select the instance you want to access.
+. Select the *Logs* tab.
+. Select the *Query log analyzer* button.
+
+Query log analyzer is split up in two parts, a timeline and a table with two tabs:
+
+* *Query timeline* - Timeline showing metrics for number of queries, failed queries and query latency.
+* *Summary* - An aggregated view of query logs, giving a high level overview over the selected time period.
+* *Details* - A detailed view showing individual query execution in the selected time period.
```
Comment (SeBBBe) [inline modules/ROOT/pages/platform/logging/query-log-analyzer.adoc:20]:
```suggestion
* *Details* - A detailed view showing individual query executions in the selected time period. 
```

File: modules/ROOT/pages/platform/logging/query-log-analyzer.adoc [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+. Select the *Logs* tab.
+. Select the *Query log analyzer* button.
+
+Query log analyzer is split up in two parts, a timeline and a table with two tabs:
+
+* *Query timeline* - Timeline showing metrics for number of queries, failed queries and query latency.
+* *Summary* - An aggregated view of query logs, giving a high level overview over the selected time period.
+* *Details* - A detailed view showing individual query execution in the selected time period. 
+
+To fetch logs first choose a time range in the Query timeline.
```
Comment (SeBBBe) [inline modules/ROOT/pages/platform/logging/query-log-analyzer.adoc:22]:
```suggestion
To fetch logs, first choose a time range in the Query timeline. 
```

File: modules/ROOT/pages/platform/logging/query-log-analyzer.adoc [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+* Last 6 hours
+* Last 24 hours
+* Last 3 days
+* Last week
+
+
+=== Zoom
+
+To zoom in to a narrower time interval, select and drag inside any chart to select your desired time interval.
+The data will automatically update to match the increased resolution.
```
Comment (SeBBBe) [inline modules/ROOT/pages/platform/logging/query-log-analyzer.adoc:46]: Maybe somehow clarify that this is true for the chart data, but not the table data

File: modules/ROOT/pages/platform/logging/query-log-analyzer.adoc [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+
+=== Sort table
+
+By default the table will be sorted on *Count* for *Summary* and *Status* for *Details*.
+To sort by a column (such as Max Time ms) click on the column heading.
+
+=== Modify columns
+
+The columns in the table can be modified by clicking the button to the right of the column row.
+Columns can be [en|dis]abled or the order changed using the grid icon at the top right of the table.
```
Comment (SeBBBe) [inline modules/ROOT/pages/platform/logging/query-log-analyzer.adoc:64]: > [en|dis]abled

I couldn't find a specific example, but I don't believe this type of short form adheres to the editorial style guide.

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-aura-382.surge.sh

New pages:
- [Query log analyzer](https://neo4j-docs-aura-382.surge.sh/aura/platform/logging/query-log-analyzer)

File: modules/ROOT/pages/platform/logging/query-log-analyzer.adoc [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+* Last 6 hours
+* Last 24 hours
+* Last 3 days
+* Last week
+
+=== Zoom
+
+To zoom in to a narrower time interval, select and drag inside any chart to select your desired time interval.
+The data in the timeline will automatically update to match the increased resolution.
+To update the table, a new request would be required with the *Fetch logs* button.
```
Comment (SeBBBe) [inline modules/ROOT/pages/platform/logging/query-log-analyzer.adoc:46]:
```suggestion
To update the table, click the *Fetch logs* button.
```

Current wording sounds a bit clunky.

File: modules/ROOT/pages/platform/logging/query-log-analyzer.adoc [hunk 6]
```diff
@@ -0,0 +1,74 @@
...
+
+To fetch logs, first choose a time range in the Query timeline. 
+With a time selection done, press the *Fetch logs* button. 
+You may optionally choose any filters or search text if required, then press *Go*.
+
+A summary of query executions is returned, showing aggregations per query.
+To see the individual query executions, click the right arrow at the end of the line to show details for that query.
+The details pane shows individual executions.
+
+== Chart interactions
```
Comment (SeBBBe) [inline modules/ROOT/pages/platform/logging/query-log-analyzer.adoc:30]: Should we mention that the entire chart can be collapsed to give more screen estate to the table? 🤔

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-aura-382.surge.sh

New pages:
- [Query log analyzer](https://neo4j-docs-aura-382.surge.sh/aura/platform/logging/query-log-analyzer)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-aura-382.surge.sh

New pages:
- [Query log analyzer](https://neo4j-docs-aura-382.surge.sh/aura/platform/logging/query-log-analyzer)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-aura-382.surge.sh

New pages:
- [Query log analyzer](https://neo4j-docs-aura-382.surge.sh/aura/platform/logging/query-log-analyzer)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-aura-382.surge.sh

New pages:
- [Query log analyzer](https://neo4j-docs-aura-382.surge.sh/aura/platform/logging/query-log-analyzer)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-aura-382.surge.sh

New pages:
- [Query log analyzer](https://neo4j-docs-aura-382.surge.sh/aura/platform/logging/query-log-analyzer)

File: modules/ROOT/pages/platform/logging/query-log-analyzer.adoc [hunk 7]
```diff
@@ -0,0 +1,103 @@
...
+
+=== Expand query
+
+In the table three rows of query text will be shown. 
+To see the whole query if the query is longer, press the *View more* button under the query text.
+
+== Limitations
+
+* Query logs are available for a period of 7 days, and each request can be for up to 24 hours of data.
+* The Query timeline may show activity from internal meta queries, which are filtered in the table.
```
Comment (SeBBBe) [inline modules/ROOT/pages/platform/logging/query-log-analyzer.adoc:102]:
```suggestion
* The query timeline may show activity from internal meta queries, which are filtered in the table.  
```

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-aura-382.surge.sh

New pages:
- [Query log analyzer](https://neo4j-docs-aura-382.surge.sh/aura/platform/logging/query-log-analyzer)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-aura-382.surge.sh

New pages:
- [Query log analyzer](https://neo4j-docs-aura-382.surge.sh/aura/platform/logging/query-log-analyzer)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-aura-382.surge.sh

New pages:
- [Query log analyzer](https://neo4j-docs-aura-382.surge.sh/aura/platform/logging/query-log-analyzer)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-aura-382.surge.sh

New pages:
- [Query log analyzer](https://neo4j-docs-aura-382.surge.sh/aura/platform/logging/query-log-analyzer)

**Comment by SeBBBe** [comment]
Already merged from my fork since I couldn't push changes to this one. Also don't have permission to close this PR so leaving this note instead.

**Comment by DavidJsson** [comment]
Closed in favour of #393

**Pull request closed without merging**